### PR TITLE
Updated test suite page

### PR DESCRIPTION
**Pull request by Big-Shark**
https://github.com/propelorm/Propel2/pull/1059

### CODE REVIEW

File: documentation/cookbook/working-with-test-suite.markdown [hunk 1]
```diff
@@ -80,11 +79,8 @@ Example:
 ```bash
 DB_HOSTNAME=192.168.0.3 DB_USER=nonroot DB_PW=topsecret ./tests/bin/setup.mysql.sh
 DB_HOSTNAME=192.168.0.3 DB_USER=postgres DB_PW=secret ./tests/bin/setup.pgsql.sh
-./tests/bin/setup.sqlite.sh #does not have any configuration ability
```
Comment (marcj) [inline documentation/cookbook/working-with-test-suite.markdown]: maybe we should add here something so people know they have to remove the sqlite file to reset the test database.

**Comment by Big-Shark** [comment]
https://github.com/propelorm/Propel2/pull/1075

**Comment by Big-Shark** [comment]
@marcj @robin850 ?

**Comment by robin850** [comment]
Thanks @Big-Shark, sorry for the delay !

**Pull request merged**